### PR TITLE
Improve dev flow

### PR DESCRIPTION
**Pull request by sleshchenko**
### What does this PR do?
The goal of this PR is to make running local Frontend and Backend easier and more straightforward.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,35 +24,32 @@ Below you can find installation instructions
 docker build . -f build/dockerfiles/Dockerfile -t quay.io/eclipse/che-dashboard:next
 ```
 
-## Running
+## Running locally against remote Che Cluster
 
-Install all dependencies:
+To run Dashboard against Che Cluster you need access to Kubernetes cluster where it lives.
+So, make sure kubeconfig from $KUBECONFIG (or if unset ~/.kube/config) has the target cluster as current.
+If no you may need to do oc logic (if it's OpenShift) or modify it manually if it's Kubernetes.
```
Comment (gorkem) [inline README.md:31]: Should be oc login

Comment (sleshchenko) [reply README.md:31]: :-D definitely!
Thanks!

File: README.md [hunk 2]
```diff
@@ -71,42 +68,16 @@ metadata:
     che.eclipse.org/ADDITIONAL_REDIRECT_URIS_env-name: ADDITIONAL_REDIRECT_URIS
     che.eclipse.org/ADDITIONAL_WEBORIGINS_env-name: ADDITIONAL_WEBORIGINS
 data:
-  ADDITIONAL_WEBORIGINS: '"http://localhost:3000"'
-  ADDITIONAL_REDIRECT_URIS: '"http://localhost:3000/*"'
   ADDITIONAL_WEBORIGINS: '"http://localhost:8080"'
```
Comment (sleshchenko) [inline README.md:71]:
```suggestion
  ADDITIONAL_WEBORIGINS: '"http://localhost:8080","http://localhost:3000"'
```

File: README.md [hunk 3]
```diff
@@ -71,42 +68,16 @@ metadata:
...
   ADDITIONAL_REDIRECT_URIS: '"http://localhost:8080/*"'
```
Comment (sleshchenko) [inline README.md:72]:
```suggestion
  ADDITIONAL_REDIRECT_URIS: '"http://localhost:8080/*","http://localhost:3000/*"'
```

File: README.md (hunk 2 above)
Comment (olexii4) [reply README.md]: I have fixed it.

File: README.md (hunk 3 above)
Comment (olexii4) [reply README.md]: I have fixed it.

File: local-start.sh [hunk 4]
```diff
@@ -1,32 +1,67 @@
...
+
+DASHBOARD_FRONTEND=packages/dashboard-frontend
+DASHBOARD_BACKEND=packages/dashboard-backend
+
+parse_args "$@"
+
+if [ "$FORCE_BUILD" == "true" ] || \
+    [ ! -d $DASHBOARD_FRONTEND/lib ] || [ -z "$(ls -A $DASHBOARD_FRONTEND/lib)" ]; then
+  echo "[INFO] Compiling frontend"
+  yarn --cwd $DASHBOARD_FRONTEND build:dev
```
Comment (sleshchenko) [inline local-start.sh:53]: it does not work for me afer rebasing

File: README.md [hunk 5]
```diff
@@ -71,40 +76,16 @@ metadata:
     che.eclipse.org/ADDITIONAL_REDIRECT_URIS_env-name: ADDITIONAL_REDIRECT_URIS
     che.eclipse.org/ADDITIONAL_WEBORIGINS_env-name: ADDITIONAL_WEBORIGINS
 data:
-  ADDITIONAL_WEBORIGINS: '"http://localhost:8080"'
-  ADDITIONAL_REDIRECT_URIS: '"http://localhost:8080/*"'
+  ADDITIONAL_WEBORIGINS: '"http://localhost:8080", "http://localhost:3000"'
```
Comment (sleshchenko) [inline README.md:79]: it does not seem to work yet. I'll take a look tomorrow

File: README.md [hunk 6]
```diff
@@ -24,35 +24,40 @@ Below you can find installation instructions
...
+yarn --cwd packages/dashboard-frontend build:watch
+```
 
-We can start dev-server for dashboard-frontend only. The start command requires to specify
-a remote Eclipse Che server like:
+As an alternative for frontend you can run Dev Server
 
 ```sh
-./local-start.sh
+cd packages/yarn start --env.server=http://localhost:8080/
```
Comment (akurinnoy) [inline README.md:55]:
```suggestion
cd packages/dashboard-frontend && yarn start --env.server=http://localhost:8080/
```

or

 ```suggestion
yarn --cwd=packages/dashboard-frontend start --env.server=http://localhost:8080/
```

File: README.md (hunk 5 above)
Comment (sleshchenko) [reply README.md:79]: Added hint that keycloak may need restart

**Review by olexii4 — APPROVED** [review]

**Comment by sleshchenko** [comment]
Happy path passed and it's just infra that delays status reporting. So, I'm merging it.

**Pull request merged**